.github/workflow-gen/Program.cs

```diff
@@ -120,7 +151,7 @@ void GenerateReleaseWorkflow(Component component)
         .Defaults().Run("bash", component.Name).Job;
 
     tagJob.Step()
-        .ActionsCheckout();
+        .ActionsCheckout("11bd71901bbe5b1630ceea73d27597364c9af683");
```

Comment:
Why do we need a particular hash version for the checkout step?

Reply:
To satisfy the SAST gods :(